src/SDK/Sdk.php

```diff
@@ -43,6 +44,23 @@ public static function isInstrumentationDisabled(string $name): bool
         return in_array($name, Configuration::getList(Variables::OTEL_PHP_DISABLED_INSTRUMENTATIONS));
     }
 
+    /**
+     * Tests whether developer mode has been enabled by config.
+     */
+    public static function isDeveloperModeEnabled(): bool
+    {
+        if (Configuration::has(Variables::OTEL_PHP_DEVELOPER_MODE_ENABLED)) {
+            return Configuration::getBoolean(Variables::OTEL_PHP_DEVELOPER_MODE_ENABLED);
+        }
+
+        try {
```

Comment:
I think it should only be enabled by variable (and default to off). Is there any benefit other than convenience to also checking InstalledVersions?

Reply:
No, purely the convenience. I will go with your lead here, will remove this dependency/complexity.